docs/userGuide/includingContents.md

```diff
@@ -62,6 +62,13 @@
         site.json
       some.md (it include a file in site A, <include src="./A/other.md">)
     ```
+    
+    You may reference files from external directories using the following prefixes:
+    ```html

```

Comment:
Don't think the html annotation is necessary for this code block (there's no HTML code inside).

Reply:
Yeah, similar to the code block just before this.